logdevice/common/NodesConfigurationPoller.h

```diff
@@ -21,7 +21,8 @@ namespace facebook { namespace logdevice {
 class ClusterState;
 
 // an instantiation of ObjectPoller which can be used to poll NodesConfiguration
-// updates from existing server nodes in NodesConfiguration
+// updates from existing server nodes in NodesConfiguration. We have implemented to use the same server
+//everytime by providing a unique seed per instance of ServerBasedNodesConfigurationManager
```

Comment:
I don't think `NodesConfigurationPoller` needs to mention `ServerBasedNodesConfigurationStore` here. Instead you'd want to describe its own behavior. So something like:

```
If the same seed is provided to the `NodesConfigurationPoller`, the same set of servers will be picked for the polling round. This can be used to implement stickiness in the polling set.
```